### ISSUE TITLE
how to get d_srgan.npz load

### ISSUE THREAD

**Issue by wl3b10s** [tensorlayer/srgan#116]
when using training stage.

i got "d_srgan.npz" not found.

there is g_srgan download link in readme, but how to get d_srgan.npz??

thanks.

**Comment by vivid-chen** [comment]
you can annotation load .npz
`# tl.files.load_and_assign_npz(sess=sess, name=checkpoint_dir + '/d_{}.npz'.format(tl.global_flag['mode']), network=net_d)`

and run main.py
d_srgan.npz will be created.

**Comment by zhaoyucong** [comment]
@wl3b10s   Can you train normally? I have the same problem

**Comment by vivid-chen** [comment]
at first， you should comment the code(#......) about loading the file d_srgan.npz.
then you can try to run the code of training.
and a d_srgan.npz will be generated by the code after the first training.


PS:sorry, my english is poor and you may understand what i mean?




------------------ 原始邮件 ------------------
发件人: "zhaoyucong"<notifications@github.com>;
发送时间: 2019年5月14日(星期二) 下午4:23
收件人: "tensorlayer/srgan"<srgan@noreply.github.com>;
抄送: "陈智勇"<1031268032@qq.com>;"Comment"<comment@noreply.github.com>;
主题: Re: [tensorlayer/srgan] how to get d_srgan.npz load (#116)




@wl3b10s   Can you train normally? I have the same problem

**Comment by zhaoyucong** [comment]
@vivid-chen  I commented the code as you said, but he still can't train.  Here are his closing tips
tensorflow 1.4.1   tensorlayer 1.8.1

 2019-05-15 16:12:17.332395: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1120] Creating TensorFlow device (/device:GPU:0) -> (device: 0, name: GeForce GTX 1080 Ti, pci bus id: 0000:1a:00.0, compute capability: 6.1)
[TL] [*] Load checkpoint/g_srgan.npz SUCCESS!
  Loading conv1_1: (3, 3, 3, 64), (64,)
  Loading conv1_2: (3, 3, 64, 64), (64,)
  Loading conv2_1: (3, 3, 64, 128), (128,)
  Loading conv2_2: (3, 3, 128, 128), (128,)
  Loading conv3_1: (3, 3, 128, 256), (256,)
  Loading conv3_2: (3, 3, 256, 256), (256,)
  Loading conv3_3: (3, 3, 256, 256), (256,)
  Loading conv3_4: (3, 3, 256, 256), (256,)
  Loading conv4_1: (3, 3, 256, 512), (512,)
  Loading conv4_2: (3, 3, 512, 512), (512,)
  Loading conv4_3: (3, 3, 512, 512), (512,)
  Loading conv4_4: (3, 3, 512, 512), (512,)
  Loading conv5_1: (3, 3, 512, 512), (512,)
  Loading conv5_2: (3, 3, 512, 512), (512,)
  Loading conv5_3: (3, 3, 512, 512), (512,)
  Loading conv5_4: (3, 3, 512, 512), (512,)
  Loading fc6: (25088, 4096), (4096,)
  Loading fc7: (4096, 4096), (4096,)
  Loading fc8: (4096, 1000), (1000,)
sample HR sub-image: (16, 384, 384, 3) -1.0 1.0
sample LR sub-image: (16, 96, 96, 3) -1.0 1.0
 ** fixed learning rate: 0.000100 (for init G)
2019-05-15 16:12:31.902147: E tensorflow/stream_executor/cuda/cuda_dnn.cc:378] Loaded runtime CuDNN library: 7103 (compatibility version 7100) but source was compiled with 7005 (compatibility version 7000).  If using a binary install, upgrade your CuDNN library to match.  If building from sources, make sure the library loaded at runtime matches a compatible version specified during compile configuration.
2019-05-15 16:12:31.903002: F tensorflow/core/kernels/conv_ops.cc:667] Check failed: stream->parent()->GetConvolveAlgorithms( conv_parameters.ShouldIncludeWinogradNonfusedAlgo<T>(), &algorithms) 
Aborted (core dumped)

**Comment by Qixiaojiu** [comment]
@zhaoyucong Can you train normally? I have the same problem. And I can't find Vgg19.npz.
